.analysis_options

```diff
@@ -59,7 +59,6 @@ linter:
 
     # === style rules ===
     - always_declare_return_types
-    - always_put_control_body_on_new_line
```

Comment:
This lint is just an enforcement of the flutter style guide rule [Separate the 'if' expression from its statement](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement).
Does plugins follow the [Dart style guide](https://www.dartlang.org/guides/language/effective-dart/style) or the [Style guide for Flutter repo](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) ?